deploy/BUILD.bazel

```diff
@@ -30,3 +31,20 @@ filegroup(
     tags = ["automanaged"],
     visibility = ["//visibility:public"],
 )
+
+filegroup(
+    name = "kustomize-base",
+    srcs = ["kustomize-base.yml"],
+    tags = ["automanaged"],
+    visibility = ["//visibility:public"],
+)
+
+# legacy kustomize base in parent is a little awkward
+modify_file(
+    name = "kustomize-base-legacy",
```

Comment:
Can we move the definition for this rule (`kustomize-legacy`) + the `kustomize-base.yaml` file into `//deploy/kustomize`, as well as the `kustomize-regular` (as `kustomize-regular`) so these are all kept together? If we go ahead with this approach the release tool will publish these two files as part of the GitHub release, meaning they'll need unique names.

Reply:
Unfortunately no.  The path is the kustomize base, you can not have two kustomize bases in the same directory(AFAIK).   Additionally, the kustomize base cannot reference resources outside of itself (up a directory), so it needs to live up the tree.   If we could move the legacy release to a parallel directory, this would look a little cleaner to me, such as:

```yaml
bases:
  - 'git@github.com/jetstack/cert-manager/releases/regular/download/v0.15.0'
```
(maybe omit regular)

AND

```yaml
bases:
  - 'git@github.com/jetstack/cert-manager/releases/legacy/download/v0.15.0'
```